src/overlays/actors/ovl_En_Yb/z_en_yb.c

```diff
@@ -37,59 +46,447 @@ const ActorInit En_Yb_InitVars = {
     (ActorFunc)EnYb_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80BFB2B0 = {
-    { COLTYPE_NONE, AT_NONE, AC_ON | AC_TYPE_ENEMY, OC1_ON | OC1_TYPE_ALL, OC2_TYPE_1, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_ON, },
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_ON | AC_TYPE_ENEMY,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0x00000000, 0x00, 0x00 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_ON,
+    },
     { 20, 40, 0, { 0, 0, 0 } },
 };
 
-#endif
-
-extern ColliderCylinderInit D_80BFB2B0;
-
-extern UNK_TYPE D_06000200;
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Yb/EnYb_Init.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Yb/EnYb_Destroy.s")
+// crashes if I try to mod it in to look at it
+//  assumption: draw uses two different skeleton functions, might be incompatible
+static AnimationHeader* gYbUnusedAnimations[] = { &object_yb_Anim_000200 };
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Yb/func_80BFA2FC.s")
+static LinkAnimationHeader* gLinkAnimations[] = { &gameplay_keep_Linkanim_00DF28, &gameplay_keep_Linkanim_00CF98 };
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Yb/func_80BFA350.s")
+static Vec3f D_80BFB2E8 = { 0.0f, 0.5f, 0.0f };
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Yb/func_80BFA444.s")
+static Vec3f D_80BFB2F4 = { 500.0f, -500.0, 0.0f };
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Yb/func_80BFA5CC.s")
+static Vec3f D_80BFB300 = { 500.0f, -500.0f, 0.0f };
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Yb/func_80BFA634.s")
+void EnYb_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnYb* this = THIS;
+    s16 tempCutscene;
+    s32 i;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Yb/func_80BFA67C.s")
+    Actor_SetScale(&this->actor, 0.01f);
+    ActorShape_Init(&this->actor.shape, 0.0f, EnYb_ActorShadowFunc, 20.0f);
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Yb/func_80BFA6E0.s")
+    // I dont know why
+    SkelAnime_InitFlex(globalCtx, &this->skelAnime, &gYbSkeleton, &object_yb_Anim_000200,
+                       (void*)(((s32) & this->limbDrawTbl) & ~0xF), (void*)((s32) & this->transitionDrawTable & ~0xF),
+                       ENYB_LIMBCOUNT);
```

Comment:
```suggestion
    SkelAnime_InitFlex(globalCtx, &this->skelAnime, &gYbSkeleton, &object_yb_Anim_000200,
                      (uintptr_t)this->limbDrawTbl & ~0xF, (uintptr_t)this->transitionDrawTable & ~0xF,
                       ENYB_LIMBCOUNT);
```
Also works, and is cleaner imo. Not sure why they did this though.